apps/summerfi-api/lib/sdk-router-function/src/procedures/deploymentsProcedure.ts

```diff
@@ -0,0 +1,10 @@
+import { publicProcedure } from '~src/trpc'
```

Comment:
@robercano this is not used anywhere rigth?

Reply:
Right, I'll remove it